docs/docs/external/getting-started/interop/asset-transfer.md

```diff
@@ -126,3 +128,182 @@ Assuming that the `simpleassettransfer` chaincode has been deployed in both netw
+  ```
+  (check token balance for `PartyA` by running the command `CORDA_PORT=10006 ./clients/build/install/clients/bin/clients get-asset-states-by-type t1`)
+- Let `PartyA` pledge these tokens in `Corda_Network` to be transferred to `PartyB` of `Corda_Network2` (pledge burns the tokens in the source/exporting network):
+  ```bash
+  CORDA_PORT=10006 ./clients/build/install/clients/bin/clients transfer pledge-asset --fungible --timeout="3600" --import-network-id='Corda_Network2' --recipient='O=PartyB, L=London, C=GB' --param='t1:5'
+  ```
+  Note the `pledge-id` displayed after successful execution of the command, which will be used in next steps. Let's denote it `<pledge-id>` which is a hexadecimal string (pledge details can be cross checked using the commands `CORDA_PORT=10006 ./clients/build/install/clients/bin/clients transfer is-asset-pledged -pid <pledge-id>` and `CORDA_PORT=10006  ./clients/build/install/clients/bin/clients transfer get-pledge-state -pid <pledge-id>`; moreover, check the token asset balance for `PartyA` in `Corda_Network` by running the command `CORDA_PORT=10006 ./clients/build/install/clients/bin/clients get-asset-states-by-type t1` which should not include the asset `t1:5` issued earlier).
+- Let `PartyB` claim in `Corda_Network2` the tokens which are pledged in the Corda network `Corda_Network` by replacing `<pledge-id>` with the above hexadecimal value (claim issues the tokens in the destination/importing network):
+  ```bash
+  NETWORK_NAME=Corda_Network2 CORDA_PORT=30009 ./clients/build/install/clients/bin/clients transfer claim-remote-asset --pledge-id='<pledge-id>' --locker='O=PartyA, L=London, C=GB' --transfer-category='token.corda' --export-network-id='Corda_Network' --param='t1:5' --import-relay-address='localhost:9082'
+  ```
+  (the `linear-id`, which is displayed after successful execution of the above command, can be used to check the newly issued tokens for `PartyB` by running `CORDA_PORT=30009 ./clients/build/install/clients/bin/clients get-state-using-linear-id <linear-id>`; or simply check the token balance for `PartyB` by running the command `CORDA_PORT=30009 ./clients/build/install/clients/bin/clients get-asset-states-by-type t1` which should output `5` tokens of type `t1`)
+
+The above steps complete a successful asset transfer from the Corda network `Corda_Network` to the Corda network `Corda_Network2`. In addition to the above commands, following is an extra option.
+
+- Let `PartyA` in `Corda_Network` try re-claim the token `t1:5` asset, which will succeed only if the asset was not claimed by `PartyB` and the pledge has expired:
+  ```bash
+  NETWORK_NAME=Corda_Network CORDA_PORT=10006 ./clients/build/install/clients/bin/clients transfer reclaim-pledged-asset --pledge-id=<pledge-id> --export-relay-address='localhost:9081' --transfer-category='token.corda' --import-network-id='Corda_Network2' --param='t1:5'
+  ```
+
+### **Transfer or recover token (Corda fungible house-token) assets**
```

Comment:
We can remove this section, as we only demonstrate corda simple app in getting started, not house token.

Reply:
Done.